assets/sass/utilities/_utilities-text.scss

```diff
@@ -81,6 +81,10 @@
     @include wrap-words();
 }
 
+.u-wrap-links a {
```

Comment:
In the spirit of being intentional where we use this I've added a new utility class which opts in to wrapping links, we can use it on a case-by-case basis like the more general `u-wrap-words`. Trying to add these styles in more global places has proven too wide reaching.